### PR TITLE
Added travis-ci configuration file.

### LINKED ISSUE #15: Replace .gitlab-ci.yml with travis.yml and CI reports to reflect hyperkitty
## Description
As a developer I am supposed to use the CI tool for the git repository management software used (github) also the CI reports should reflect the systers clone of hyperkitty and not that from it's [main development repository.
](https://gitlab.com/mailman/hyperkitty)

## Acceptance Criteria
- [x] add and configure.travis.yml file
- [x] update CI reports to reflect Systers clone of hyperkitty (this repo)

## Definition of Done
- [x] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
4 hours

### PR DESCRIPTION
**Pull request by Fenn-CS**
### Description
Added `.travis.yml` file with basic configurations to enable this repository use github's own CI server.

Fixes #15 

### Type of Change:
- Code
- Quality Assurance

### How Has This Been Tested?
Tested with travis-ci live server at `https://travis-ci.org/Fenn-CS/hyperkitty`

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+sudo: false
+language: python
+
+python:
+  - "3.4"
+
+install:
+  - pip install whoosh
```
Comment (Monal5031) [inline .travis.yml]: Separate requirements.txt?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  - pip install mock
+  - pip install BeautifulSoup4
+before_script:
+  - python setup.py develop
+script:
+  - python example_project/manage.py test --settings=hyperkitty.tests.settings_test
```
Comment (Monal5031) [inline .travis.yml]: tox file is already added, we can use that directly?

Comment (Fenn-CS) [reply .travis.yml]: I have not used tox before? Can you show me an example? Thanks... Will it make it simpler?

File: .travis.yml (hunk 1 above)
Comment (Fenn-CS) [reply .travis.yml]: Are you asking if there's a `requirements.txt`? or suggesting we add one?

Comment (Monal5031) [reply .travis.yml]: Yeah a separate file.

File: .travis.yml (hunk 2 above)
Comment (Monal5031) [reply .travis.yml]: Yeah I'll link you a PR where I used the tox file in travis. Also after using tox file you won't have to add req.txt.

Comment (Monal5031) [reply .travis.yml]: https://github.com/systers/postorius/pull/22
@Fenn-CS

File: .travis.yml (hunk 1 above)
Comment (Fenn-CS) [reply .travis.yml]: I have added tox... as you specified and it has solved the issue of having to add a requirements file.

File: .travis.yml (hunk 2 above)
Comment (Fenn-CS) [reply .travis.yml]: Thanks... Tox is now employed.

**Comment by MehaKaushik** [comment]
Hey @Fenn-CS any particular reason, why you didn't merge this PR ? If not, then please go ahead and merge it.

**Comment by ExactlyMay** [comment]
@MehaKaushik I squashed and merged it. Hope this helps.

**Pull request merged**